libsolidity/parsing/DocStringParser.cpp

```diff
@@ -39,12 +39,22 @@ string::const_iterator skipLineOrEOS(
 	return (_nlPos == _end) ? _end : ++_nlPos;
 }
 
-string::const_iterator firstSpaceOrTab(
+string::const_iterator firstNonIdentifier(
 	string::const_iterator _pos,
 	string::const_iterator _end
 )
 {
-	return boost::range::find_first_of(make_pair(_pos, _end), " \t");
+	// Identifier start
+	auto currPos = _pos;
```

Comment:
Perhaps use `currPos == _pos` to check for first character and just use a single loop?

Reply:
Utilizes `liblangutil/Common.h` now. I think it's clearer what happens.